src/pages/BenefitPackagePage.js

```diff
@@ -0,0 +1,110 @@
+import React, { useEffect } from 'react';
+import { connect, useDispatch } from 'react-redux';
+import {
+  Form,
+  formatMessageWithValues,
+  useHistory,
+  useModulesManager,
+} from '@openimis/fe-core';
+import { bindActionCreators } from 'redux';
+import { injectIntl } from 'react-intl';
+import { withTheme, withStyles } from '@material-ui/core/styles';
+import BenefitPackageTabPanel from '../components/BenefitPackageTabPanel';
+import BenefitPackagePlanPanel from '../components/BenefitPackagePlanPanel';
+import { RIGHT_BENEFICIARY_SEARCH } from '../constants';
+import BenefitPackageIndividualPanel from '../components/BenefitPackageIndividualPanel';
+import { fetchBeneficiary, fetchBenefitPlan } from '../actions';
+
+const styles = (theme) => ({
+  page: theme.page,
+});
+
+function BenefitPackagePage({
+  rights,
+  intl,
+  classes,
+  beneficiaryUuid,
+  beneficiary,
+  fetchedBeneficiary,
+  benefitPlanUuid,
+  benefitPlan,
+  fetchedBenefitPlan,
+}) {
+  const history = useHistory();
+  const modulesManager = useModulesManager();
+  const dispatch = useDispatch();
+  const dependenciesFetched = fetchedBeneficiary && fetchedBenefitPlan;
+
+  const back = () => history.goBack();
+
+  useEffect(() => {
```

Comment:
I wonder if we can shortcut to such statement this useEffect: 
```
useEffect(() => {
  beneficiaryUuid && dispatch(fetchBeneficiary(modulesManager, { beneficiaryUuid }));
  benefitPlanUuid && dispatch(fetchBenefitPlan(modulesManager, [`id: "${benefitPlanUuid}"`]));
}, [beneficiaryUuid, benefitPlanUuid]);
```
Just let me know if this works or not. If no, please keep current solution. It is just the idea of improvement and reducing the if statements to reduce complexity.

Reply:
I will keep it like it is since eslint shows this message: _Expected an assignment or function call and instead saw an expression.eslintno-unused-expressions_. Moreover, in my opinion those if statements are super clean and readable.

Reply:
ok, then let's keep your current approach